bedrock/thunderbird/templates/thunderbird/organizations.html

```diff
@@ -0,0 +1,44 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "thunderbird/base-resp.html" %}
+
+{% block page_title %}{{ _('Thunderbird Extended Release Support for Your Organization, Business, Enterprise - Overview') }}{% endblock %}
+
+{% block body_id %}thunderbird-organizations{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'thunderbird-organizations' %}
+{% endblock %}
+
+{% block content %}
+  <header id="main-feature" class="stacked">
+    <h1>{{ _('Mozilla Thunderbird <span class="large">Extended Support Release</span>') }}</h1>
+    <h4>{{ _('A community-lead project that allows organizations to benefit from the speed, flexibility and security of Thunderbird while getting the support they need.') }}</h4>
```

Comment:
I wonder about this `h4` directly after the `h1`. Perhaps it would be a better call to use a paragraph element here? It reads more like a paragraph rather than a sub-heading.